docs/tutorials/release-using-channels.md

```diff
@@ -0,0 +1,55 @@
+<a name="release_using_channels"></a>
+# Release Using Channels / Auto-Updates With Channels
+
+## Description
+Channels are useful to distribute "beta" or "alpha" releases of your application to a chosen set of users. This allows to test an application before release it as "stable".
+
+Users which receive "beta" version will get "stable" versions too. Otherwise, users who don't want "beta" will only get "stable" releases.
+
+They are tree channels level ordered by stability:
```

Comment:
typo. i think you meant to say "There are three channels, ordered by stability"

Reply:
Thanks ! Yes this is exactly what I'm trying to say ;)